frontend/src/composables/useChainSelection.ts

```diff
@@ -9,20 +9,8 @@ import type { SelectorOption } from '@/types/form';
 export function useChainSelection(
   provider: Ref<IEthereumProvider | undefined>,
   chains: Ref<ChainConfigMapping>,
+  selectedSourceChain: Ref<SelectorOption<Chain> | null>,
```

Comment:
As this composable gets now used within two components, I think the more suitable adaption would be to just make this a filter. So say `ignoreChains: Ref<Array<Chain>>` (or without an array) or something like that (filter, ... idk). So this composable should not care that there is something like source and target or whatever. It just allows you the convenience to filter the chains. For what ever reason you like...

Reply:
Was great idea. Looks much cleaner now! :heavy_check_mark: